app/src/neptune_eye/config.py

```
@@ -0,0 +1,223 @@
+    if config.model.override_model_path is not None:
+        root_dir = find_project_root()
+        model_path = root_dir / config.model.override_model_path
+        if not model_path.exists():
+            raise ValueError(f"Override model path does not exist: {model_path}")
+    
+    # Validate movie path if using movie input
+    if config.input.source == InputSource.MOVIE:
+        root_dir = find_project_root()
+        movie_path = root_dir / config.input.movie_path
+        if not movie_path.exists():
+            raise ValueError(f"Movie file does not exist: {movie_path}")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)